cmd/cluster/deployment/burst_1.1.go

```diff
@@ -34,12 +34,6 @@ func init() {
 	astarteBurstProfile11.Compatibility.MaxAstarteVersion, _ = semver.NewVersion("1.1.99")
 	astarteBurstProfile11.Compatibility.MinAstarteVersion, _ = semver.NewVersion("1.1.0")
 
-	// Let components burst only
```

Comment:
We can leave a comment here, something like "Do not set component's resources to make sure we're setting a burst profile".